packages/app/src/debug/LayeredBrowserIcons.vue

```diff
@@ -33,7 +33,7 @@ import { IconBrowserChrome,
 // Note: These browser names should map to the list of logoPaths found at https://github.com/cypress-io/cypress-services/blob/46073cb5c387af3b203404280a1664a85e233b93/packages/common/src/logos/getLogoPath.ts
 // Some of these are currently incorrect (ex: CHROME-CANARY).
 // We are also missing browsers from the list as well
```

Comment:
Is this comment still relevant?

Reply:
Updated to use the stable icons for the situations where we didn't have one for the specific browser channels. Also, added a generic globe icon for when an icon is not found.